mne/beamformer/_dics.py

```diff
@@ -66,9 +66,12 @@ def _apply_dics(data, info, tmin, forward, noise_csd, data_csd, reg,
 
     Cm = data_csd.data
 
-    # Calculating regularized inverse, equivalent to an inverse operation after
-    # regularization: Cm += reg * np.trace(Cm) / len(Cm) * np.eye(len(Cm))
-    Cm_inv = linalg.pinv(Cm, reg)
+    # Tikhonov regularization using reg parameter to control for trade-off between spatial resolution and noise sensitivity
```

Comment:
this line is too long. mne has pep8 tests and they allow only lines <= 79 characters